### PR TITLE
INTMDB-364 Add support for serverless private endpoints (AWS) 

### PR DESCRIPTION
**Pull request by martinstibbe**
## Description

Add support for serverless private endpoints (AWS) 

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update
- [ ] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [x] I have run make fmt and formatted my code

### CODE REVIEW

**Comment by andreaangiolillo** [comment]
I think that this PR contains some changes that we don't want to merge such as 
[Adding a github actions to automatically close stale issues/PRs based…](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/909/commits/a86484d65fea7d060de7697e02a718b303c05515)

**Review by andreaangiolillo — COMMENTED** [review]
I will review it once the lint errors have been addressed

File: website/docs/guides/1.5.0-upgrade-guide.html.markdown [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+description: |-
+MongoDB Atlas Provider 1.5.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.5.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.5.0 has a number of new and exciting features and changes.
+
+New Features:
+* You can now manage AWS Serverless Private Endpoints [`mongodbatlas_serverless_private_endpoints`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_private_endpoints)
```
Comment (Zuhairahmed) [inline website/docs/guides/1.5.0-upgrade-guide.html.markdown:14]:
```suggestion
* You can now manage private endpoints for Atlas serverless instances on AWS [`mongodbatlas_privatelink_endpoint_severless`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_serverless), [`mongodbatlas_privatelink_endpoint_service_severless`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/privatelink_endpoint_service_serverless)
```

**Review by Zuhairahmed — CHANGES_REQUESTED** [review]
Few comments below, also can you add documentation as part of this PR? i think we will need 5 
**Resource:** 
- mongodbatlas_privatelink_endpoint_severless
- mongodbatlas_privatelink_endpoint_service_severless 

**Data Source:** 
- mongodbatlas_privatelink_endpoint_severless
- mongodbatlas_privatelink_endpoint_service_severless
- mongodbatlas_privatelink_endpoints_service_severless

**Comment by Zuhairahmed** [comment]
@themantissa can you review as well after @andreaangiolillo and me are okay as one last sanity check, will ping you when ready

**Comment by martinstibbe** [comment]
Ok that was from rebasing from master as staging branch was oldSent from my iPhoneOn Nov 9, 2022, at 10:16 AM, Andrea Angiolillo ***@***.***> wrote:﻿
I think that this PR contains some changes that we don't want to merge such as
Adding a github actions to automatically close stale issues/PRs based…

**Pull request closed without merging**